helper/header.html

```diff
@@ -36,9 +36,6 @@
   TeX: { equationNumbers: { autoNumber: "AMS" } }
 });
 </script>
```

Comment:
This mathjax config is still valid with the “—mathjax” option, right?

Reply:
Yes. It has been tested using `jem-press/content/latex.md`.